content/r/concepts/vectors/vectors.md

```diff
@@ -0,0 +1,108 @@
+---
+Title: 'Vectors'
+Description: 'Vectors are used to store and use multiple data values of the same type.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Vectors'
+  - 'Variables'
+  - 'Integers'
+  - 'Booleans'
+  - 'Strings'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+Vectors are used to store and use multiple data values of the same type that can be accessed later by subsequent code. In R, the indexing of the vectors will start from ‘1’ and not from ‘0’ unlike the other programming languages.
+
+## Creating a Vector
+
+In R, vectors are created or initialised using the ‘c’ function which means combining different elements together.
+Vectors can be of different datatypes.
```

Comment:
This should be rephrased to avoid confusion. As is, it may be read that a single vector can contain multiple datatypes.

Reply:
This still needs to be addressed.